src/spanningtrees/boruvka.jl

```diff
@@ -0,0 +1,56 @@
+@traitfn function boruvka_mst(graph::AG::(!IsDirected), 
```

Comment:
We need a docstring for this function. As far as I know, this does not work with the `@traitfn` macro, but you can create a separate dummy function `function boruvka_mst end` and place the docstring above this definition.

Reply:
I'd also like to rename `graph` to `g`. This is consistent with other functions in LG.